### ISSUE TITLE
produce test set using cd-hit representative sequences

### ISSUE THREAD

**Issue by philippmuench** [philippmuench/HiddenGenome#4]
- run http://weizhongli-lab.org/cd-hit/ on crispr_full at e.g. 80 percent
- create training dataset only containing cluster centroids
- remove these sequences from test dataset
- add option to train() function to supply additional training set
